primitives/historical-data/README.md

```diff
@@ -0,0 +1,18 @@
+## Historical data
+
+Crate with functionality to manage data that stores its own history.
+
+This covers:
+- linear history driven data, eg. transactional layers for overlay.
+- long term storage with multiple branch, eg. offchain storage.
+
+General design is container where query and update requires global
```

Comment:
```suggestion
General design is to be a container where query and update both require a global
```

i'm changing this from non-english to english. i've no idea if what i'm writing is what you were meaning @cheme .